docs/versioning.rst

```diff
@@ -0,0 +1,145 @@
+**********
+Versioning
+**********
+
+
+Current Release Stages
+======================
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Runtimes
+     - Minimum Trace Agent
+     - Stage
+   * - 0.x
+     - CPython: 2.7, 3.5–3.10
+     - 
+     - MAINTENANCE [2022-03-01]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.28
+     - CURRENT
+
+
+Runtimes
+========
+
+
+.. list-table:: Supported runtime versions
```

Comment:
this table isn't rendering for me in GitHub preview, but the one above is, I can't tell what is wrong with it (if anything is)

Reply:
same with OS